WORKING-GROUPS.md

```diff
@@ -73,7 +73,7 @@ Each working group has one or more leads which coordinate the activities of the
 <img width="30px" src="https://avatars1.githubusercontent.com/u/623453?s=400&v=4">   | [John Howard](https://github.com/howardjohn)       | Google     | Networking
 &nbsp;                                                                               | [Steven Landow](https://github.com/stevenctl)      | Solo.io    | Networking - Control Plane
 &nbsp;                                                                               | [Costin Manolache](https://github.com/costinm)     | Google     | Environments
-&nbsp;                                                                               | [Sam Naser](https://github.com/monkeyanator)       | Google     | Environments
+&nbsp;                                                                               | [Sam Naser](https://github.com/monkeyanator)       | Aviatrix     | Environments
```

Comment:
Remove the extra space.

Reply:
https://github.com/istio/community/pull/1310